src/services/swap_service.ts

```diff
@@ -237,8 +238,18 @@ export class SwapService {
     private _convertSourceBreakdownToArray(
         sourceBreakdown: SwapQuoteOrdersBreakdown,
     ): GetSwapQuoteResponseLiquiditySource[] {
+        const defaultSourceBreakdown: SwapQuoteOrdersBreakdown = Object.keys(ERC20BridgeSource).reduce(
+            (acc: SwapQuoteOrdersBreakdown, k: any): SwapQuoteOrdersBreakdown => {
+                return {
+                    ...acc,
+                    [k]: ZERO,
+                };
+            },
+            {},
+        );
```

Comment:
Optional nit:

So I feel like `reduce()` is an anti-pattern because it's never obvious what it's doing, and is usually pretty long-winded. Might I suggest...?
```ts
const defaultSourceBreakdown = Object.assign(...Object.values(ERC20BridgeSource).map(s => ({ [s]: ZERO )}));
```